src/psfmachine/machine.py

```diff
@@ -167,6 +167,7 @@ def __init__(
         self.cartesian_knot_spacing = "sqrt"
         # disble tqdm prgress bar when running in HPC
         self.quiet = False
+        self.cont_mag_limit = None
```

Comment:
What would `cont` mean here? Maybe we can come up with a more descriptive name since I think there will be two magnitude limits and I'm worried it will be confusing.

Astronomy would tell us to use `hard_mag_limit` and `soft_mag_limit` or something but I think that's also not great

Reply:
supposed to mean contaminant, but yes it isn't intuitive. I think just `contaminant_mag_limit`. Though I want to have it as an internal variable, not a method argument, in that case the log warning message is useful. But, it can still be modified if users want to adjust the `uncontaminated_pixel_mask`.

Reply:
This sounds perfect!